platform-hub-api/app/models/kubernetes_cluster.rb

```diff
@@ -26,6 +27,13 @@ class KubernetesCluster < ApplicationRecord
   validates :description, :s3_region, :s3_bucket_name, :s3_object_key,
             :s3_access_key_id, :s3_secret_access_key, presence: true
 
+  validates :aws_account_id,
```

Comment:
shouldn't this be a required field?

Reply:
Not necessarily; also, can't make it required as it will affect existing records.

Reply:
Sure. Just thought it's something we know upfront usually so could make it a required field.

Reply:
We _could_ make it required from now on, but…
a) We wouldn't be able to add a non-NULL db constraint (which is not ideal), due to existing records (unless we defaulted to some value).
b) Even if we didn't do (a) we could still make it so when you next update a cluster you have to provide the account ID… but I can imagine a situation where someone is trying to update some other important value (like the S3 bucket details) and it becomes frustrating because they have to dig up the AWS account ID in that moment,.

Reply:
OK, we can leave as is and take care of it later. I guess accounts without ID won't be taken into account for costing so that's OK for now.